### ISSUE TITLE
'atomic uninstall' is broken

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic#986]
Using git master, running `atomic uninstall` returns `[Errno 13] Permission denied`

We saw this in our automated tests on the [Fedora Atomic Host Continuous](https://pagure.io/fedora-atomic-host-continuous) stream, but I was able to replicate it with git master on F25 Cloud, too.

I think it was introduced with 1938ec31086f80b0baa9097e24a1849e21b71b52

See below:

```
# git rev-parse HEAD
e93b8319ad31977de504a9a6f1318e03ea64b861
# ./atomic pull docker.io/cockpit/ws:latest                                                                                                                                         
Pulling docker.io/cockpit/ws:latest ...
Copying blob sha256:2bf01635e2a0f7ed3800c8cb3effc5ff46adc6b9b86f0e80743c956371efe553
 59.01 MB / 69.50 MB [================================================>--------]
Copying blob sha256:fb6a56efc47a17b2e39fb81a1428d8cd946182d9ea94661130f985094519e6fa
 0 B / 13.95 KB [--------------------------------------------------------------]
Copying blob sha256:7f0ddc461820674c34d1a8fc353bc21b959951ea90909f8ad292a9d9f15c067b
 0 B / 335 B [-----------------------------------------------------------------]
Copying blob sha256:6ae135e249a3816988c73f743af7860d84cfd9247251ad65c19e5761f957c770
 113.14 MB / 114.47 MB [=======================================================]
Copying config sha256:6c148ec7fa4f308e10c6f3ad89a5a1aa1679b927ab49b89ced2cd284568957b2
 0 B / 5.27 KB [---------------------------------------------------------------]
Writing manifest to image destination
Storing signatures
 5.27 KB / 5.27 KB [===========================================================]
# ./atomic images list
   REPOSITORY             TAG      IMAGE ID       CREATED            VIRTUAL SIZE   TYPE      
   docker.io/cockpit/ws   latest   6c148ec7fa4f   2017-04-23 14:39   552.92 MB      docker    

# ./atomic install docker.io/cockpit/ws:latest
/usr/bin/docker run --rm --privileged -v /:/host docker.io/cockpit/ws:latest /container/atomic-install
+ sed -e /pam_selinux/d -e /pam_sepermit/d /etc/pam.d/cockpit
+ mkdir -p /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ chmod 755 /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ chown root:root /host/etc/cockpit/ws-certs.d /host/etc/cockpit/machines.d
+ mkdir -p /host/var/lib/cockpit
+ chmod 775 /host/var/lib/cockpit
+ chown root:wheel /host/var/lib/cockpit
+ mkdir -p /etc/ssh
+ /bin/mount --bind /host/etc/cockpit /etc/cockpit
+ /usr/sbin/remotectl certificate --ensure
# ./atomic run docker.io/cockpit/ws:latest
/usr/bin/docker run -d --privileged --pid=host -v /:/host docker.io/cockpit/ws:latest /container/atomic-run --local-ssh

This container uses privileged security switches:

INFO: --pid=host 
      Processes in this container can see and interact with all processes on the host and disables SELinux within the container.

INFO: --privileged 
      This container runs without separation and should be considered the same as root on your system.

For more information on these switches and their security implications, consult the manpage for 'docker run'.

49120e46bcd7202cda7ffed65bf4484fcb2211d5cf9c7e071be1083c87b6651b
# ./atomic containers list
   CONTAINER ID IMAGE                COMMAND              CREATED          STATE      BACKEND    RUNTIME   
   49120e46bcd7 docker.io/cockpit/ws /container/atomic-ru 2017-05-01 17:49 running    docker     docker    
# ./atomic stop 49120e46bcd7
# ./atomic containers list                                                                                                                                                          
# ./atomic uninstall docker.io/cockpit/ws:latest
/ u s r / b i n / d o c k e r r u n - - r m - - p r i v i l e g e d - v / : / h o s t I M A G E / c o n t a i n e r / a t o m i c - u n i n s t a l l
[Errno 13] Permission denied
# ./atomic --debug uninstall docker.io/cockpit/ws:latest                                                                                                                            
Namespace(_class=<class 'Atomic.uninstall.Uninstall'>, args=[], assumeyes=False, debug=True, display=False, force=False, func='uninstall', ignore=False, image='docker.io/cockpit/ws:latest', name=None, opt1=None,
 opt2=None, opt3=None, profile=False, storage=None)
/ u s r / b i n / d o c k e r r u n - - r m - - p r i v i l e g e d - v / : / h o s t I M A G E / c o n t a i n e r / a t o m i c - u n i n s t a l l
[Errno 13] Permission denied
Traceback (most recent call last):
  File "./atomic", line 203, in <module>
    sys.exit(_func())
  File "/root/atomic/Atomic/uninstall.py", line 62, in uninstall
    be.uninstall(img_obj, name=self.args.name, atomic=self, ignore=self.args.ignore)
  File "/root/atomic/Atomic/backends/_docker.py", line 471, in uninstall
    return util.check_call(cmd, env=atomic.cmd_env())
  File "/root/atomic/Atomic/util.py", line 147, in check_call
    return subprocess.check_call(cmd, env=env, stdin=stdin, stderr=stderr, stdout=stdout, close_fds=True)
  File "/usr/lib64/python2.7/subprocess.py", line 181, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib64/python2.7/subprocess.py", line 168, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib64/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib64/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 13] Permission denied
```

**Comment by baude** [comment]
Can we close this ?

**Comment by miabbott** [comment]
It looks like I said this was closed in #987, so I'll close it.
